ext/ddtrace_profiling_native_extension/profiling.c

```diff
@@ -4,7 +4,7 @@
 
 static VALUE native_working_p(VALUE self);
 
-void Init_ddtrace_profiling_native_extension(void) {
+void __attribute__ ((visibility ("default"))) Init_ddtrace_profiling_native_extension(void) {
```

Comment:
for conciseness I would have a `#define EXPORT __attribute__ ((visibility ("default")))` somewhere (ir whatever fits better than `EXPORT`. Also, doesn't Ruby have such define for native extensions?

Reply:
Good question! There's a `RUBY_FUNC_EXPORTED` but only from Ruby 3.0 on 😭 

Since there's no common header where to place it, for now I left a `#define DDTRACE_EXPORT` in `profiling.c`, will move it once there's other places where we want to use it.